azure-pipelines.yml

```diff
@@ -320,6 +336,21 @@ jobs:
 
 # Scheduled runs will run all of the jobs on the PerfTigers, as well as jobs on the PerfSnakes and the Arm64 job
 - ${{ if and(ne(variables['System.TeamProject'], 'public'), in(variables['Build.Reason'], 'Schedule')) }}:
+
+  # Windows x64 SDK scenario benchmarks
+  - template: /eng/performance/scenarios.yml
+    parameters:
+      osName: windows
+      osVersion: RS5
+      architecture: x64
+      pool: Hosted VS2017
+      kind: sdk_scenarios
+      queue: Windows.10.Amd64.19H1.Tiger.Perf
```

Comment:
@DrewScoggins  Maybe you'd want to double-check if the queue is right.

Reply:
Looks right to me.